FoxHound.App/Blogs/UpdateBlog/UpdateBlogCommandValidator.cs

```diff
@@ -0,0 +1,21 @@
+﻿using FluentValidation;
+
+namespace FoxHound.App.Blogs.UpdateBlog
+{
+    public class UpdateBlogCommandValidator : AbstractValidator<UpdateBlogCommand>
```

Comment:
Since this validator and the add validator are pretty much identical, it might be worth abstracting it into it's own common blog validator. Consider doing this as a separate PR, or we could create an issue for it, and do it at some later date.

Reply:
I had the same thoughts, it should be a separate PR - I can handle this next.

Reply:
Issue created, assigned to myself.